libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -242,30 +242,56 @@ def ListMIGSByInstanceName(self, zone: str) -> Dict[str, str]:
         groups_by_instance[instance.name] = group_id
     return groups_by_instance
 
-  def ListMIGS(self, zone: str) -> Dict[str, List['GoogleComputeInstance']]:
+  def ListMIGS(self, location: str) -> Dict[str, List['GoogleComputeInstance']]:
     """Gets the managed instance groups in a particular zone.
 
     Returns a dictionary, with as keys the managed instance groups, and as
     values a list of instances belonging to the group.
 
     Args:
-      zone (str): The zone in which to list managed instance groups.
+      location (str): The region/zone in which to list managed instance groups.
 
     Returns:
       Dict[str, List[GoogleComputeInstance]]: A mapping from managed instance
           groups to their managed GCE instances.
     """
     groups_client = self.GceApi().instanceGroupManagers() # pylint: disable=no-member
-    responses = common.ExecuteRequest(
-        groups_client, 'list', {
-            'project': self.project_id,
-            'zone': zone,
-        })
-
     groups = defaultdict(list)
+    try:
```

Comment:
instead of doing this, maybe you can use a regex to determine if the location is a zone, I did similar before, this would work well:
```
ZONE_REGEX = re.compile(
    r'\A(?P<zone>(?P<region>(?P<geo_tag>[a-zA-Z0-9]+)-[a-zA-Z0-9]+)'
    r'-[a-zA-Z0-9]+)\Z'
)
```

Reply:
Updated, thanks.